tests/0102-static_group_rebalance.c

```diff
@@ -177,7 +177,8 @@ int main_0102_static_group_rebalance (int argc, char **argv) {
         /* 3x heartbeat interval to give time for c[0] to recognize rebalance */
         rd_sleep(9);
 
-        rd_kafka_flush(c[0].rk, 5000);
+        do_consume(c[0].rk, 5/*5s*/);
```

Comment:
Since the function name is not suitable for what we are doing here I suggest you add a comment saying: Wait for rebalance, or similar.